clusterProof/main.go

```diff
@@ -0,0 +1,27 @@
+package clusterProof
```

Comment:
Is this still used? (there's another main.go, and this isn't package main, so I bet it doesn't become a binary)